docs/server-side/v2/api-reference/policy/update-a-template.mdx

```diff
@@ -214,3 +214,8 @@ curl --location --request POST 'https://api.100ms.live/v2/templates/6316d5ca5a26
 | settings <br/>                 | `object`  | Modified settings. Since this is a POST request, you need to provide the whole settings object to modify/persist/override it. <br/><br/>check the [settings object arguments section](/server-side/v2/policy/create-template-via-api#settings-object) for the list of arguments supported              | Yes      |
 | name <br/>                     | `string`  | If you don't provide the `name`, earlier provided name is used. Trying to modify a template's name to match another existing template's name produces overwrite error.                                                                                                                                 | No       |
 | default <br/>                  | `boolean` | Whether this is default template for customer. The default template for a customer is used when no template is provided in create room API.                                                                                                                                                            | No       |
+
```

Comment:
please move this higher in the doc - it's easy to miss at the end

Reply:
also, can this be made more obvious with pseudocode?

```
1. get the template object with the get API (link to it)
2. modify the template object to update it
3. post the updated object as request body with this API
```

Reply:
Made the changes as suggested.